internal/commands/vaultsecrets/secrets/helper/helper.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package helper
+
+import (
+	"context"
+	"fmt"
+
+	preview_secret_service "github.com/hashicorp/hcp-sdk-go/clients/cloud-vault-secrets/preview/2023-11-28/client/secret_service"
+	preview_models "github.com/hashicorp/hcp-sdk-go/clients/cloud-vault-secrets/preview/2023-11-28/models"
+	"github.com/hashicorp/hcp/internal/commands/vaultsecrets/secrets/appname"
+	"github.com/hashicorp/hcp/internal/pkg/cmd"
+	"github.com/posener/complete"
+)
+
+// PredictAppName returns a predict function for application names.
+func PredictSecretName(ctx *cmd.Context, orgID, projectID string, client preview_secret_service.ClientService) complete.PredictFunc {
+	return func(args complete.Args) []string {
+		if len(args.Completed) > 1 {
+			return nil
+		}
+
+		appName := appname.Get()
```

Comment:
`appname.Get()` is yielding empty as I have a feeling at this time the `vault-secrets secrets` command's `PersistentPreRun` has not run (maybe ?). checking with @dadgar